.github/workflows/build.yaml

```diff
@@ -31,6 +31,10 @@ jobs:
           # see https://github.com/golangci/golangci-lint-action/issues/119#issuecomment-981090648 for output
           args: --config=.golangci.yml --verbose --out-${NO_FUTURE}format colored-line-number
           skip-cache: true
+      - name: Run govulncheck
+        run: |
+          go install golang.org/x/vuln/cmd/govulncheck@latest
+          govulncheck ./...
```

Comment:
should we use https://github.com/golang/govulncheck-action ?

Reply:
Ah, nice! Last I checked, there was no GHA yet.